lib/diplomat/token.rb

```diff
@@ -38,11 +38,15 @@ def read(id, options = {}, not_found = :reject, found = :return)
 
     # List all the ACL tokens
     # @param policy [String] filters the token list matching the specific policy ID
+    # @param role [String] filters the token list matching the specific role ID
```

Comment:
could you provide an example of such filter in doc?

Reply:
Here is a direct link to the filters: https://www.consul.io/api/acl/tokens.html#parameters-5